plugins/woocommerce/src/Internal/Admin/ReviewsListTable.php

```diff
@@ -211,10 +211,41 @@ protected function column_date( $item ) {
 	/**
 	 * Renders the product column.
 	 *
-	 * @param object|array $item Review or reply being rendered.
+	 * @see WP_Comments_List_Table::column_response() for consistency.
+	 *
+	 * @return void
 	 */
-	protected function column_response( $item ) {
-		// @TODO Implement in MWC-5337 {agibson 2022-04-12}
+	protected function column_response() {
+		$product_post = get_post();
+
+		if ( ! $product_post ) {
+			return;
+		}
+
+		?>
+		<div class="response-links">
+			<?php
+
+			if ( current_user_can( 'edit_post', $product_post->ID ) ) :
```

Comment:
```suggestion
			if ( current_user_can( 'edit_product', $product_post->ID ) ) :
```

Should we use this instead?

Reply:
I'm fine with that -- `manage_woocommerce` would be probably too restrictive?